docs/presentation/Hierarchies/ChildNodeRule.md

```diff
@@ -1,34 +1,115 @@
-| *Branch content attributes* |
-| `specifications`            | No        | [`ChildNodeSpecification[]`](./index.md#specifications)              | `[]`    | Specifications that define what content the rule returns.                                                                                                                                  |
-| `customizationRules`        | No        | [`CustomizationRule[]`](../Customization/index.md#rules)             | `[]`    | Customization rules that are applied for the content returned by this rule.                                                                                                                |
-| `subConditions`             | No        | `SubCondition[]`                                                     | `[]`    | Specifies child node rules which are only used when specific condition is satisfied                                                                                                        |
-
-## Example
-
-```JSON
-{
-  "ruleType": "ChildNodes",
-  "condition": "ParentNode.IsOfClass(\"Model\", \"BisCore\")",
-  "requiredSchemas": [{ "name": "BisCore", "minVersion": "1.0.1" }],
-  "priority": 999,
-  "stopFurtherProcessing": true,
-  "specifications": []
-}
+| Name                                                        | Required? | Type                                              | Default |
+| ----------------------------------------------------------- | --------- | ------------------------------------------------- | ------- |
+| *Placement attributes*                                      |
+| [`condition`](#attribute-condition)                         | No        | [ECExpression](./ECExpressions.md#rule-condition) | `""`    |
+| [`requiredSchemas`](#attribute-requiredschemas)             | No        | `RequiredSchemaSpecification[]`                   | `[]`    |
+| [`priority`](#attribute-priority)                           | No        | `number`                                          | `1000`  |
+| [`onlyIfNotHandled`](#attribute-onlyifnothandled)           | No        | `boolean`                                         | `false` |
+| [`stopFurtherProcessing`](#attribute-stopfurtherprocessing) | No        | `boolean`                                         | `false` |
+| *Branch content attributes*                                 |
+| [`specifications`](#attribute-specifications)               | No        | `ChildNodeSpecification[]`                        | `[]`    |
+| [`customizationRules`](#attribute-customizationrules)       | No        | `CustomizationRule[]`                             | `[]`    |
+| [`subConditions`](#attribute-subconditions)                 | No        | `SubCondition[]`                                  | `[]`    |
+
+### Attribute: `condition`
+
```

Comment:
The word "Attribute" is repeated in each heading. Take inspiration from [webpack documentation](https://webpack.js.org/configuration/optimization/#optimizationminimize) and put plain attribute names for headings.

Reply:
I actually like the prefix - it makes it clear that the section contains attribute specification and not something else. Especially since it's likely for this page to receive some content that's not an attribute documentation.

BTW, webpack also uses a prefix, just a different one. In our case we, if we followed webpack's example, we should prefix with `ChildNodeRule.`